config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -203,3 +203,56 @@ periodics:
     testgrid-tab-name: dualstack-azure-e2e-master
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: "Dual-stack e2e tests on a Kubernetes cluster provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud"
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure-dualstack
+    decorate: true
+    decoration_config:
+      timeout: 2h
+    always_run: false
+    optional: true
+    path_alias: k8s.io/kubernetes
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-dind-enabled: "true"
```

Comment:
I think that this is for kind or others that use docker in docker  , let's ask @BenTheElder

Reply:
yep, I'm not sure what this job uses though.

Reply:
same for the security context below. if docker is not in use, it shouldn't be included. if it is, this is expected